interface/main/main_screen.php

```diff
@@ -414,17 +414,20 @@ function generate_html_end()
     //display the php file containing the password expiration message.
     $frame1url = "pwd_expires_alert.php?csrf_token_form=" . attr_url(CsrfUtils::collectCsrfToken());
     $frame1target = "adm";
+    $frame1label = "";
 } elseif (!empty($_POST['patientID'])) {
     $patientID = 0 + $_POST['patientID'];
     if (empty($_POST['encounterID'])) {
         // Open patient summary screen (without a specific encounter)
         $frame1url = "../patient_file/summary/demographics.php?set_pid=" . attr_url($patientID);
         $frame1target = "pat";
+        $frame1label = xl('Patient Search/Add Screen');
     } else {
         // Open patient summary screen with a specific encounter
         $encounterID = 0 + $_POST['encounterID'];
         $frame1url = "../patient_file/summary/demographics.php?set_pid=" . attr_url($patientID) . "&set_encounterid=" . attr_url($encounterID);
         $frame1target = "pat";
+        $frame1label = xl('Patient Search/Add Screen');
```

Comment:
I'm unsure but I think xl('Patient Search/Add Screen') will make translation engine burp!
'Search / Add Screen' maybe not but xl('Patient Search') . ' / ' . xl('Add Screen') definitely translates.

@bradymiller can you clarify for us?

Reply:
The `/` is ok
Actually better to just keep it as one `Patient Search/Add Screen` constant like you have. Since some languages may use the `/` character differently in their languages.

Reply:
https://github.com/openemr/openemr/blob/a6abf755f97ad984835e4de9d5e0792c9f31ed25/library/globals.inc.php#L189
Should i change here too?